config/locales/client.en.yml

```diff
@@ -97,7 +97,7 @@ en:
       comment: "comment"
       share: "Share"
       thought: "thought"
-      on: "on"
+      "on": "on"
```

Comment:
on, off, yes, no must be wrapped in quote >.<
Source: https://medium.com/@morgler/getting-a-translation-missing-for-off-in-rails-9b6518f21838